docs/source/community/index.md

```diff
@@ -1,15 +1,20 @@
 # Community
 
 `movement` is made possible by the generous contributions of [many people](target-people).
+
 We welcome and encourage contributions in any form—whether it is fixing a bug,
-developing a new feature, or improving the documentation.
+developing a new feature, or improving the documentation—as long as you follow our
+[code-of-conduct](movement-github:#coc-ov-file).
```

Comment:
Should we have code-of-conduct on the website too rather than pointing to github?

Reply:
You're right! option 1 involves adding another step into the makefiles to fetch the file and then using myst to include the file. but since the file is rarely updated we could just keep it simple and copy the contents over.